README.md

```diff
@@ -93,24 +93,30 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
 1. Install tools specified in the previous section (Docker, kubectl, skaffold)
 
-1. Create a Google Kubernetes Engine cluster and make sure `kubectl` is pointing
+2. Create a Google Kubernetes Engine cluster and make sure `kubectl` is pointing
    to the cluster.
 
         gcloud services enable container.googleapis.com
 
         gcloud container clusters create demo --enable-autoupgrade \
-            --enable-autoscaling --min-nodes=3 --max-nodes=10 --num-nodes=5
+            --enable-autoscaling --min-nodes=3 --max-nodes=10 --num-nodes=5 \
+            -- scopes https://www.googleapis.com/auth/cloud_debugger
```

Comment:
there's an extra space between `--` and `scopes`.

Reply:
you might wanna delete this space.

Reply:
Haha. Sneaky one!